<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>417</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>389</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ex1:Reverse string with UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>404</width>
      <height>264</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainlayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="vowelcheck_Button">
         <property name="text">
          <string>vowelcheck</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="palindrome_Button">
         <property name="text">
          <string>palindrome</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="OutputLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="LatinPigButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>LatinPigGame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ConfirmButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Reverse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExitButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="OutputLine"/>
     </item>
     <item row="5" column="0">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item row="6" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="InputLine"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="InputLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Input</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QTextEdit" name="wordstatisc_txtbox"/>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="wordstatisc_Button">
       <property name="text">
        <string>单词统计</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOther">
    <property name="title">
     <string>other</string>
    </property>
    <addaction name="actionTextedit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOther"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTextedit">
   <property name="text">
    <string>textedit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
